package.json

```diff
@@ -40,5 +41,8 @@
   "bugs": {
     "url": "https://github.com/Chalarangelo/30-seconds-of-code/issues"
   },
-  "homepage": "https://github.com/Chalarangelo/30-seconds-of-code#readme"
+  "homepage": "https://github.com/Chalarangelo/30-seconds-of-code#readme",
+  "devDependencies": {
```

Comment:
Merge this with the other `devDependencies` property higher up in your package, it causes our Travis to fail right now.